claims.json

```diff
@@ -2672,8 +2672,9 @@
   "org.apache.kafka *": "apache/kafka",
   "org.apache.kudu kudu-spark2_2.11": "scalacenter/scaladex-void",
   "org.apache.kudu kudu-spark_2.10": "scalacenter/scaladex-void",
-  "org.apache.logging.log4j log4j-api-scala_2.10": "apache/logging-log4j2",
-  "org.apache.logging.log4j log4j-api-scala_2.11": "apache/logging-log4j2",
+  "org.apache.logging.log4j log4j-api-scala_2.10": "apache/logging-log4j-scala",
+  "org.apache.logging.log4j log4j-api-scala_2.11": "apache/logging-log4j-scala",
+  "org.apache.logging.log4j log4j-api-scala_2.12": "apache/logging-log4j-scala",
```

Comment:
Got it. Then there is an error in the documentation. https://github.com/apache/logging-log4j-scala#usage It should be like this:

```scala
libraryDependencies ++= Seq(
  "org.apache.logging.log4j" %% "log4j-api-scala" % "11.0",
  "org.apache.logging.log4j" % "log4j-api" % "2.8.2",
  "org.apache.logging.log4j" % "log4j-core" % "2.8.2" % Runtime
)
```

Java project with one `%`